src/main/java/com/bhoruka/bloodbank/model/response/SearchCampResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.bhoruka.bloodbank.model.response;
+
+import java.util.Date;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class SearchCampResponse {
+
+    String id;
+
+    String partnerId;
+
+    Date dateOfCamp;
+
+    Long expectedNoOfDonors;
+
+    Long actualNoOfDonors;
+
+    Integer pageNo;
+
+    Integer pageSize;
```

Comment:
Why is this required in the response?

Reply:
Won't we have to show the page number?

Reply:
but this is pageSize, which is being passed by the client to begin with.